build.sh

```diff
@@ -42,7 +46,7 @@ build()
     echo "Building software in '${BUILD_DIR}'."
     cmake \
         -DCPACK_PACKAGE_VERSION="${RELEASE_VERSION:-}" \
-        -DCMAKE_C_COMPILER="${CROSS_COMPILE}gcc" \
+        -DCMAKE_C_COMPILER="aarch64-linux-gnu-gcc" \
```

Comment:
Why is the compiler now hardcoded in comparison to the parameterized older version?

Reply:
In other repositories, we could have different compilers, but in this case, it is impossible. This tool will hardly ever change after today, therefore I decided to simplify it.